src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
```

Comment:
Typically App (and the App Test) would go in the Containers folder.  This is because it's considered one of the highest level components.

Reply:
It's a bit confusing. I want to research more before making any changes here.

Reply:
Here's a pretty good resource https://daveceddia.com/react-project-structure/

Reply:
> src/components - All your Presentational (aka Dumb) components go here. These are the simple stateless ones that just take props.

`App` looks pretty `dumb` to mee :smile:

Reply:
haha, although then most of the files would be in the container folder at this point.   I'm thinking of this more as Parent and Children.  I cringe at the thought because it's pretty unusual to send the Parent component (CardList) down to a Child component to then be exported back up 2 directories to index.js.   It makes me wonder if extracting App is really necessary at this point since it could technically still live in index.js due to it's (dumbness). Take a look back at the structure I had with Main and index.   Unless we are going to give some state to App, I'm leaning toward it going back into index.  @jhowardjr what are your thoughts on this one?

Reply:
I’ll take a look tomorrow. angular-cli projects have something  similar, but they put the app component on the root of src folder.

Reply:
I’ll take a look tomorrow. angular-cli projects have something  similar, but they put the app component on the root of src folder.

Reply:
CardList is in `containers` just because it was there before I renamed it from Main and I agree that it's awkward right now. App component might go to `root` and I would like to keep it in case we decide to add some description, footer, [<your_idea_here>]...